common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -39,23 +41,27 @@ def setUp(self):
         self.manual_cohort_id = self.add_manual_cohort(self.course_fixture, self.manual_cohort_name)
 
         # create a non-instructor who will be registered for the course and in the manual cohort.
-        self.student_name = "student_user"
+        self.student_name = "student_" + str(uuid.uuid4().hex)[:12]
+        self.student_email = self.student_name + "@example.com"
         self.student_id = AutoAuthPage(
```

Comment:
Maybe there should be a helper for authorizing with a unique user, where the function takes a user name prefix or something. That could be used three times here, and I imagine many times in other tests.